File: dataLayer/entities/notification.py
from sqlalchemy import Column, String, Integer, Date

from dataLayer.database_connection import Base


class Notification(Base):
    __tablename__ = 'Notification'

    id = Column('Id', Integer, primary_key=True)
    message = Column('Message', String)

    def __init__(self, message):
        self.message = message

File: DataLayer/Entities/Notification.cs
using DataLayer.Entities.Common;

namespace DataLayer.Entities
{
    public class Notification : EntityBase
    {
        public string Message { get; set; }
        public ImageAttachment Image { get; set; }
        public int NotificationTypeId { get; set; }
        public NotificationType NotificationType { get; set; }
    }
}